chia/wallet/trade_manager.py

```diff
@@ -507,8 +507,12 @@ async def _create_offer_for_ids(
                         wallet = await self.wallet_state_manager.get_wallet_for_asset_id(asset_id.hex())
                     if not callable(getattr(wallet, "get_coins_to_offer", None)):  # ATTENTION: new wallets
                         raise ValueError(f"Cannot offer coins from wallet id {wallet.id()}")
+                    # For the XCH wallet also include the fee amount to the coins we use to pay this offer
+                    get_coin_amount = abs(amount)
+                    if wallet.type() == WalletType.STANDARD_WALLET:
+                        get_coin_amount += fee
                     coins_to_offer[id] = await wallet.get_coins_to_offer(
-                        asset_id, uint64(abs(amount)), min_coin_amount, max_coin_amount
+                        asset_id, uint64(get_coin_amount), min_coin_amount, max_coin_amount
```

Comment:
`get_coin_amount` sounds like a function, maybe rename to `total_coin_amount_with_fee` or something similar?

Reply:
What about 72d90e4a5411e8e8383e8149d97c7634d9c74315?